staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go

```diff
@@ -201,6 +207,13 @@ func TooLong(field *Path, value interface{}, maxLength int) *Error {
 	return &Error{ErrorTypeTooLong, field.String(), value, fmt.Sprintf("must have at most %d characters", maxLength)}
 }
 
+// TooMany returns a *Error indicating "too many". This is used to
+// report that a given list has too many items. This is similar to TooLong,
+// but the returned error indicates quantity instead of length.
+func TooMany(field *Path, value interface{}, maxQuantity int) *Error {
```

Comment:
We can't put value in the error - it's too big!  Instead pass the len() and store that.